pkg/dicomio/reader.go

```diff
@@ -94,6 +94,9 @@ func NewReader(in *bufio.Reader, bo binary.ByteOrder, limit int64) Reader {
 }
 
 func (r *reader) BytesLeftUntilLimit() int64 {
+	if r.limit < 0 {
+		return 10000
```

Comment:
why 10000? you can always return math.MaxInt64 if needed. In the future we can probably remove this API.

Reply:
the 10k figure was just some rough estimation for the max length of a DICOM header...
I'll change it to math.MaxInt64 no prob, but I think that in the future we might also have some option to read only header - which will stop reading when getting to the pixel data, saving on traffic.
we are going to process a huge amount of DICOM files, so not reading the pixel data from the buckets is going to be beneficial...